cloud/google/cmd/gce-machine-controller/app/controller.go

```diff
@@ -61,7 +62,18 @@ func StartMachineController(server *options.MachineControllerServer, shutdown <-
 		glog.Fatalf("Could not create config watch: %v", err)
 	}
 
-	actuator, err := google.NewMachineActuator(server.KubeadmToken, client.ClusterV1alpha1().Machines(corev1.NamespaceDefault), configWatch)
+	sshUser, err := ioutil.ReadFile(server.SSHUserPath)
```

Comment:
Three comments:

1. It seems odd that the ssh user is being supplied via file. It would be easier to use if it could just be a string parameter.
2. Supplying an ssh user is now required even though supplying a private/public keys is not -- is this desirable? This seems like something that should have a reasonable default but not be a required parameter.